src/regions/ca/info/faq/32_transmission-symptoms.md

```diff
@@ -1,8 +1,9 @@
 ## Is it possible to transmit COVID-19 if you do not have any symptoms (cough, fever, or difficult breathing)?
 
-People are thought to be most contagious when they are most symptomatic (the sickest).
-Some spread might be possible before people show symptoms; there have been reports of this occurring with this new coronavirus, but this is not thought to be the main way the virus spreads.
+People are thought to be more contagious when they have the most severe symptoms (when they are sickest). However, it is possible to be contagious after being exposed to COVID-19 for up to 14 days after exposure without having any symptoms, or having symptoms that can be mistaken for the flu or a common cold.
+
+Experts are currently studying in more details the transmission of the virus. Although contamination without symptoms appears to be less potent, it is not to be neglected, and isolation is therefore essential for proven contacts, as well as social distancing for all other people.
 
 More information here:
 
-- [CDC](https://www.cdc.gov/coronavirus/2019-ncov/prepare/transmission.html)
+-[CDC](https://www.cdc.gov/coronavirus/2019-ncov/prepare/transmission.html) -[Government of Canada](https://www.canada.ca/en/public-health/services/diseases/2019-novel-coronavirus-infection/symptoms.html)
```

Comment:
I'd recommend putting the second source on a new line here as well :)

Reply:
Here again, my mistake :p. Should be fixed now!